packages/api-client/src/layouts/App/ApiClientApp.vue

```diff
@@ -40,9 +40,26 @@ onBeforeMount(() => addScalarClassesToHeadless())
 const handleKeyDown = (ev: KeyboardEvent) =>
   handleHotKeyDown(ev, events.hotKeys, { hotKeys })
 
+const handleHotKey = (event?: HotKeyEvent) => {
+  if (!event) return
+
+  // We prevent default on open command so we can use it on the web
+  if (event.openCommandPalette) {
+    event.openCommandPalette.preventDefault()
```

Comment:
Do we need to prevent default here?

Reply:
yea I believe it will launch the default chrome command otherwise(search google)

Reply:
N opens a new tab for me. K and return work for the app commands though

Reply:
I think search / omnibar is CMD+L

Reply:
CTRL+K 😄 
![image](https://github.com/user-attachments/assets/52a0a52b-9c78-4643-950c-07e5fab42302)

Reply:
maybe command L for you apple nerds

Reply:
Welp

Reply:
is it actually command L? whats command K do then?

Reply:
ah okay windows has both ctrl K and ctrl E, most sites override ctrl k but you still got e, and yes apple is confirmed L so it doesn't matter. Keep the prevent default!